### ISSUE TITLE
Toolbar disappears when panning the map on MacOS Safari

### ISSUE THREAD

**Issue by hoddelhs** [nrenner/brouter-web#569]
I use BRouter with my Mac Book and iPad.  I have the problem that when moving the map, the left toolbar disappears again and again.  it is covered by the map and rarely reappears.  Unfortunately, you can't plan like that.  On the Mac I can help myself by installing the Edge Browser, so I don't have this problem.  Unfortunately, this doesn't work on the iPad. 

https://user-images.githubusercontent.com/108179387/175771250-ad1a57e8-5243-48b9-aa8f-ac24d3650602.mp4

**Comment by mjaschen** [comment]
I see that you're using bikerouter.de. Can you reproduce this issue on https://brouter.de/brouter-web/ as well? (Just to eliminate the possibility the error is specific for bikerouter.de)

**Comment by hoddelhs** [comment]
yes with BRouter 0.17.0 and the new version 0.18.0 I also have these problems. It looks like it only occurs at a certain zoom level

**Comment by stesie** [comment]
I've also noticed this a few times in the past (on DuckDuckGo Browser & Safari on iOS, both iPhone and iPad).  But never really had a look, since I (also) thought it was a problem with my instance only :)

Having dug into it a bit w/ current version on brouter.de/brouter-web/ instance ... this seems to be a z-index issue.  If brouter-web is in that particular state and you select elements from the dev tool, it can be seen that it's indeed at the right place.  Yet there are elements in `.leaflet-tile-pane` and `.leaflet-marker-pane` that have higher z-index's.

Adding a manual/explicit `z-index: 99999` style to both `.leaflet-control-container` and all its children, immediately makes the elements re-appear.

**Comment by stesie** [comment]
Bug Report w/ Webkit: https://bugs.webkit.org/show_bug.cgi?id=238589 (there's a fix, but it seems not to be included in latest iOS)

Leaflet has it as well as mentions a work-around: https://github.com/Leaflet/Leaflet/issues/8068
